hycom/RELO/src_2.2.98ZA-07Tsig0-i-sm-sse_relo_mpi/mod_hycom_fabm.F90

```diff
@@ -730,41 +780,78 @@ subroutine hycom_fabm_update(m, n, ibio)
 ! HIGH VALUES ARE SET TO A BIT LOWER VALUES TO KEEP THE MODEL REPEATEDLY
 ! EXCEEDING THE 99TH PERCENTILE
 ! THE SAME THING IS ALSO DONE AFTER THE BOTTOM EXCHANGE
```

Comment:
@caglartac you should add a comment here as to why this has been commented out

Reply:
It is commented out because it will be removed.

Reply:
OK, but if the whole thing will be removed, should you not also remove this comment?
! HIGH VALUES ARE SET TO A BIT LOWER VALUES TO KEEP THE MODEL REPEATEDLY
! EXCEEDING THE 99TH PERCENTILE
! THE SAME THING IS ALSO DONE AFTER THE BOTTOM EXCHANGE

Reply:
ok

The only change that needs to be reviewed is in lines 1045-1065 which ensures mass balance. The rest is just a print and I will remove them.

Reply:
@caglartac these lines look good to me, but I will not approve the merging to develop until after you have removed all the print-statements.

Reply:
Ok I will remove them

Reply:
@lilleannette I was more thinking that we would use this branch to test the model before pushing to the develop. That is why the prints are there, just in case I need them again if we encounter something with TP2/5

Reply:
@caglartac great, yes, we leave it like this for now, then I may also test it in TP2.